src/main/webapp/script.js

```diff
@@ -49,3 +50,160 @@ function verifyCredentials() {
     return false;
   });
 }
+
+function getGCalendarEvents(calendar, startTime, endTime) {
+  verifyCredentials().then(validCredential => {
+    if (validCredential) {
+      fetch("/events/gcalendar?start=" + startTime + "&end=" + endTime).then(response => response.json()).then(events => {
+        var fullcalendarEvents = [];
+        events.forEach(event => {
+          var start, end;
+          if (event.start.dateTime) {
+            start = event.start.dateTime.value;
+            end = event.end.dateTime.value;
+          } else {
+            start = event.start.date.value;
+            end = event.end.date.value;
+          }
```

Comment:
Could you please clarify, what is this?

Reply:
Sorry for this, but it's not my fault. Google Calendar API has two methods of storing the time and you do not know which one is used. You need to verify in which field the time is held.